mono/metadata/threads.c

```diff
@@ -5588,6 +5605,23 @@ mono_threads_detach_coop (gpointer cookie, gpointer *dummy)
 	}
 }
 
+/*
+ * mono_threads_detach_coop: called by native->managed wrappers
+ *
+ *  - @orig: the original domain which needs to be restored, or NULL.
+ *  - @dummy:
+ *    - blocking mode: contains gc unsafe transition cookie
+ *    - non-blocking mode: contains random data
+ *    - a pointer to stack, used for some checks
+ */
+void
+mono_threads_detach_coop (gpointer orig, gpointer *dummy)
+{
+	MonoStackData stackdata;
+	mono_stackdata_init (&stackdata, dummy, NULL);
```

Comment:
Why not passing `__func__` here?

Reply:
Fair enough.

Reply:
At which point I suggest further reduction, in the next commit.